docs/examples/basecoin/cmd/basecli/main.go

```diff
@@ -72,35 +72,33 @@ func main() {
 
 	// add query/post commands (custom to binary)
 	rootCmd.AddCommand(
-		client.GetCommands(
-			stakecmd.GetCmdQueryValidator(storeStake, cdc),
-			stakecmd.GetCmdQueryValidators(storeStake, cdc),
-			stakecmd.GetCmdQueryValidatorUnbondingDelegations(storeStake, cdc),
-			stakecmd.GetCmdQueryValidatorRedelegations(storeStake, cdc),
-			stakecmd.GetCmdQueryDelegation(storeStake, cdc),
-			stakecmd.GetCmdQueryDelegations(storeStake, cdc),
-			stakecmd.GetCmdQueryPool(storeStake, cdc),
-			stakecmd.GetCmdQueryParams(storeStake, cdc),
-			stakecmd.GetCmdQueryUnbondingDelegation(storeStake, cdc),
-			stakecmd.GetCmdQueryUnbondingDelegations(storeStake, cdc),
-			stakecmd.GetCmdQueryRedelegation(storeStake, cdc),
-			stakecmd.GetCmdQueryRedelegations(storeStake, cdc),
-			slashingcmd.GetCmdQuerySigningInfo(storeSlashing, cdc),
-			authcmd.GetAccountCmd(storeAcc, cdc),
-		)...)
+		stakecmd.GetCmdQueryValidator(storeStake, cdc),
+		stakecmd.GetCmdQueryValidators(storeStake, cdc),
+		stakecmd.GetCmdQueryValidatorUnbondingDelegations(storeStake, cdc),
```

Comment:
isn't this missing `getCmdQueryValidatorDelegations`  ?

Reply:
Seems was missing already. Can you open an issue for that please? It's out of scope here

Reply:
Done, thanks